WooCommerce/Classes/Authentication/AuthenticationManager.swift

```diff
@@ -109,22 +112,46 @@ class AuthenticationManager: Authentication {
                                                     LoginPrologueViewController(isFeatureCarouselShown: isFeatureCarouselShown),
                                                 statusBarStyle: .default)
 
+        let getStartedInstructions: String = {
+            if isSimplifiedLoginI1Enabled {
+                return AuthenticationConstants.getStartedInstructionsForSimplifiedLogin
+            } else if isWPComSignupEnabled {
+                return AuthenticationConstants.getStartedInstructionsWithWPComSignupEnabled
+            } else {
+                return AuthenticationConstants.getStartedInstructions
+            }
+        }()
+
+        let continueWithWPButtonTitle: String = {
+            if isSimplifiedLoginI1Enabled {
+                return AuthenticationConstants.loginButtonTitle
+            }
+            return AuthenticationConstants.continueWithWPButtonTitle
```

Comment:
nit: might be simpler

```suggestion
            return isSimplifiedLoginI1Enabled ? AuthenticationConstants.loginButtonTitle: AuthenticationConstants.continueWithWPButtonTitle
```

Reply:
Updated in 54be2c2f66163406e4146987ebca585357baff3a.